servatrice/src/servatrice.cpp

```diff
@@ -627,7 +628,7 @@ void Servatrice::statusUpdate()
 
             while (servDbSelQuery->next()) {
                 const QString userName = servDbSelQuery->value(0).toString();
-                const QString emailAddress = servDbSelQuery->value(1).toString();
+                const auto emailAddress = EmailParser::getParsedEmailAddress(servDbSelQuery->value(1).toString());
```

Comment:
We'd need a migration script to support that, as there's no guarantee the DB is "clean" since we never did any kind of cleanup after the initial parser was added